lib/python/temporal/core.py

```diff
@@ -688,13 +712,28 @@ def init(raise_fatal_error=False, skip_db_version_check=False):
                              "%(info)s") % ({"backup": backup_howto,
                                              "api": get_tgis_version(),
                                              "info": get_database_info_string()}))
-            if "tgis_db_version" in entry and entry[1] != str(get_tgis_db_version()):
-                msgr.fatal(_("Unsupported temporal database: version mismatch."
-                             "\n %(backup)sSupported temporal database version"
-                             " is: %(tdb)i\nCurrent temporal database info:"
-                             "%(info)s") % ({"backup": backup_howto,
-                                             "tdb": get_tgis_version(),
-                                             "info": get_database_info_string()}))
+        # temporal database version check
+        tgis_db_version_meta = get_tgis_db_version_from_metadata(metadata)
+        if tgis_db_version_meta != tgis_db_version:
+            message = _(
+                "Temporal database version mismatch detected.\n{backup}"
+                "Supported temporal database version is: {tdb}\n"
+                "Your existing temporal database version: {ctdb}\n"
+                "Current temporal database info: {info}".format(
+                    backup=backup_howto,
+                    tdb=tgis_db_version,
+                    ctdb=tgis_db_version_meta,
+                    info=get_database_info_string(),
+                )
+            )
+
+            msgr.fatal(
+                _(
+                    "The format of your actual temporal database is "
```

Comment:
_your actual temporal_: _current_? _existing_? or just _your temporal_?

Reply:
Changed to `existing` by 3d854e0e1